wallpaper-changer/requirements.txt

```diff
@@ -0,0 +1,7 @@
+Dependencies:-
+
+- Schedule Library
+  Installation:
+  `pip install schedule`
+
+   
```

Comment:
improve it.

Reply:
use python standard requirements.txt way. check requirement.txt file example below.
 
https://github.com/uber/Python-Sample-Application/blob/master/requirements.txt